setup.py

```diff
@@ -22,7 +22,7 @@ def read(fname):
     license='MIT',
     url='https://github.com/hackebrot/poyo',
     description='A lightweight YAML Parser for Python',
-    long_description=read('README.rst'),
+    long_description=read('README.md'),
```

Comment:
Can you please also add the following [long_description_content_type](https://packaging.python.org/guides/distributing-packages-using-setuptools/#description):

```suggestion
    long_description=read('README.md'),
    long_description_content_type="text/markdown",
```